### ISSUE TITLE
ImportError: No module named mutagen on KUBUNTU 20.4

### ISSUE THREAD

**Issue by Hannes-III** [exaile/exaile#720]
I have downloaded and installed Exaile 4.0.2 on Kubuntu 20.04.1

### Steps to Reproduce (for bugs)
1. install exaile:
```
make
sudo make install
```

2. run exaile
I had several issues and already installed some of the missing python libraries:
- python-gobject
- python-dbus
- python3-mutagen

3. re run exaile
I tried all the following without any success or valuable information about the error:
```
exaile
exaile --debug
exaile --startgui
exaile --start-anyway
```

### Expected Behavior
I can listen to Music!

### Current Behavior
I get the following error:
```
INFO    : Loading Exaile 4.0.2...
INFO    : Using Python 2.7.18rc1
INFO    : Using PyGObject 3.36.0
INFO    : Loading settings...
INFO    : Using Locale de_AT UTF-8
ERROR   : Unhandled exception
Traceback (most recent call last):
File "/usr/local/lib/exaile/xl/main.py", line 575, in __init__
self.__init()
File "/usr/local/lib/exaile/xl/main.py", line 643, in __init
from xlgui.widgets.info import Splash
File "/usr/local/lib/exaile/xlgui/__init__.py", line 40, in <module>
from xl import common, player, providers, settings, version, xdg
File "/usr/local/lib/exaile/xl/player/__init__.py", line 38, in <module>
from . import queue
File "/usr/local/lib/exaile/xl/player/queue.py", line 34, in <module>
from xl import common, event, playlist, settings
File "/usr/local/lib/exaile/xl/playlist.py", line 50, in <module>
from xl import common, dynamic, event, main, providers, settings, trax, xdg
File "/usr/local/lib/exaile/xl/dynamic.py", line 33, in <module>
from xl.trax import search
File "/usr/local/lib/exaile/xl/trax/__init__.py", line 34, in <module>
from xl.trax.track import Track
File "/usr/local/lib/exaile/xl/trax/track.py", line 39, in <module>
from xl import event, metadata, settings
File "/usr/local/lib/exaile/xl/metadata/__init__.py", line 31, in <module>
from xl.metadata._base import BaseFormat, CoverImage, NotWritable, NotReadable
File "/usr/local/lib/exaile/xl/metadata/_base.py", line 36, in <module>
import mutagen
ImportError: No module named mutagen
```

### Environment
This is the list of Files installed by packge python3-mutagen_1.44.0-1_all.deb which is the currently available package in ubuntu:
```
./
./usr/
./usr/bin/
./usr/bin/mid3cp
./usr/bin/mid3iconv
./usr/bin/mid3v2
./usr/bin/moggsplit
./usr/bin/mutagen-inspect
./usr/bin/mutagen-pony
./usr/lib/
./usr/lib/python3/
./usr/lib/python3/dist-packages/
./usr/lib/python3/dist-packages/mutagen/
./usr/lib/python3/dist-packages/mutagen/__init__.py
./usr/lib/python3/dist-packages/mutagen/_compat.py
./usr/lib/python3/dist-packages/mutagen/_constants.py
./usr/lib/python3/dist-packages/mutagen/_file.py
./usr/lib/python3/dist-packages/mutagen/_senf/
./usr/lib/python3/dist-packages/mutagen/_senf/__init__.py
./usr/lib/python3/dist-packages/mutagen/_senf/_argv.py
./usr/lib/python3/dist-packages/mutagen/_senf/_compat.py
./usr/lib/python3/dist-packages/mutagen/_senf/_environ.py
./usr/lib/python3/dist-packages/mutagen/_senf/_fsnative.py
./usr/lib/python3/dist-packages/mutagen/_senf/_print.py
./usr/lib/python3/dist-packages/mutagen/_senf/_stdlib.py
./usr/lib/python3/dist-packages/mutagen/_senf/_temp.py
./usr/lib/python3/dist-packages/mutagen/_senf/_winansi.py
./usr/lib/python3/dist-packages/mutagen/_senf/_winapi.py
./usr/lib/python3/dist-packages/mutagen/_tags.py
./usr/lib/python3/dist-packages/mutagen/_tools/
./usr/lib/python3/dist-packages/mutagen/_tools/__init__.py
./usr/lib/python3/dist-packages/mutagen/_tools/_util.py
./usr/lib/python3/dist-packages/mutagen/_tools/mid3cp.py
./usr/lib/python3/dist-packages/mutagen/_tools/mid3iconv.py
./usr/lib/python3/dist-packages/mutagen/_tools/mid3v2.py
./usr/lib/python3/dist-packages/mutagen/_tools/moggsplit.py
./usr/lib/python3/dist-packages/mutagen/_tools/mutagen_inspect.py
./usr/lib/python3/dist-packages/mutagen/_tools/mutagen_pony.py
./usr/lib/python3/dist-packages/mutagen/_util.py
./usr/lib/python3/dist-packages/mutagen/_vorbis.py
./usr/lib/python3/dist-packages/mutagen/aac.py
./usr/lib/python3/dist-packages/mutagen/ac3.py
./usr/lib/python3/dist-packages/mutagen/aiff.py
./usr/lib/python3/dist-packages/mutagen/apev2.py
./usr/lib/python3/dist-packages/mutagen/asf/
./usr/lib/python3/dist-packages/mutagen/asf/__init__.py
./usr/lib/python3/dist-packages/mutagen/asf/_attrs.py
./usr/lib/python3/dist-packages/mutagen/asf/_objects.py
./usr/lib/python3/dist-packages/mutagen/asf/_util.py
./usr/lib/python3/dist-packages/mutagen/dsf.py
./usr/lib/python3/dist-packages/mutagen/easyid3.py
./usr/lib/python3/dist-packages/mutagen/easymp4.py
./usr/lib/python3/dist-packages/mutagen/flac.py
./usr/lib/python3/dist-packages/mutagen/id3/
./usr/lib/python3/dist-packages/mutagen/id3/__init__.py
./usr/lib/python3/dist-packages/mutagen/id3/_file.py
./usr/lib/python3/dist-packages/mutagen/id3/_frames.py
./usr/lib/python3/dist-packages/mutagen/id3/_id3v1.py
./usr/lib/python3/dist-packages/mutagen/id3/_specs.py
./usr/lib/python3/dist-packages/mutagen/id3/_tags.py
./usr/lib/python3/dist-packages/mutagen/id3/_util.py
./usr/lib/python3/dist-packages/mutagen/m4a.py
./usr/lib/python3/dist-packages/mutagen/monkeysaudio.py
./usr/lib/python3/dist-packages/mutagen/mp3/
./usr/lib/python3/dist-packages/mutagen/mp3/__init__.py
./usr/lib/python3/dist-packages/mutagen/mp3/_util.py
./usr/lib/python3/dist-packages/mutagen/mp4/
./usr/lib/python3/dist-packages/mutagen/mp4/__init__.py
./usr/lib/python3/dist-packages/mutagen/mp4/_as_entry.py
./usr/lib/python3/dist-packages/mutagen/mp4/_atom.py
./usr/lib/python3/dist-packages/mutagen/mp4/_util.py
./usr/lib/python3/dist-packages/mutagen/musepack.py
./usr/lib/python3/dist-packages/mutagen/ogg.py
./usr/lib/python3/dist-packages/mutagen/oggflac.py
./usr/lib/python3/dist-packages/mutagen/oggopus.py
./usr/lib/python3/dist-packages/mutagen/oggspeex.py
./usr/lib/python3/dist-packages/mutagen/oggtheora.py
./usr/lib/python3/dist-packages/mutagen/oggvorbis.py
./usr/lib/python3/dist-packages/mutagen/optimfrog.py
./usr/lib/python3/dist-packages/mutagen/smf.py
./usr/lib/python3/dist-packages/mutagen/tak.py
./usr/lib/python3/dist-packages/mutagen/trueaudio.py
./usr/lib/python3/dist-packages/mutagen/wavpack.py
./usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/
./usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/PKG-INFO
./usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/dependency_links.txt
./usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/entry_points.txt
./usr/lib/python3/dist-packages/mutagen-1.44.0.egg-info/top_level.txt
./usr/share/
./usr/share/doc/
./usr/share/doc/python3-mutagen/
./usr/share/doc/python3-mutagen/README.Debian
./usr/share/doc/python3-mutagen/changelog.Debian.gz
./usr/share/doc/python3-mutagen/copyright
./usr/share/man/
./usr/share/man/man1/
./usr/share/man/man1/mid3cp.1.gz
./usr/share/man/man1/mid3iconv.1.gz
./usr/share/man/man1/mid3v2.1.gz
./usr/share/man/man1/moggsplit.1.gz
./usr/share/man/man1/mutagen-inspect.1.gz
./usr/share/man/man1/mutagen-pony.1.gz
```
I am using Python 3.8.2

**Comment by rokm** [comment]
> I am using Python 3.8.2

The log says otherwise:

> INFO    : Using Python 2.7.18rc1

The last released version of `exaile` (4.0.2) is still python2-only. To use `exaile` with python3, you will need to use the git master.

**Comment by Hannes-III** [comment]
> The log says otherwise
Oooops, true.

> exaile with python3, will need git master
I just tried it, all working fine. Thank you.

For others who are interested, those are the packages I had to install so that exaile could start:
-     python-gobject
-     python-dbus
-     python3-mutagen
-     python3-bsddb3
